.gitignore

```diff
@@ -1,4 +1,6 @@
 _site
 .sass-cache
 .jekyll-metadata
-@_MATERIAL
\ No newline at end of file
+@_MATERIAL
+Gemfile
+Gemfile.lock
```

Comment:
Why are you adding these to .gitignore? I think they are needed for GH Actions to build the site for pages.